### DISCUSSION TITLE
Feature Request: Smart QiYi Timer bluetooth timer option

### DISCUSSION THREAD

**Discussion by Ceqell** [cs0x7f/cstimer#311] [Ideas]
Hello, I would like to suggest a new feature for the CStimer website. I recently bought the Smart QiYi Timer, which is a bluetooth-enabled timer that can start the inspection automatically when the sensor pads are slightly tapped. I think it would be great if CStimer could support this timer and allow users to connect this timer on the website. I hope you will consider adding this option to the website. Thanks for your hard work!

**Comment by cs0x7f** [comment]
An employee of qiyi told me that the data from qiyi timer is encrypted, and they don't want to make the timer connect with cstimer wirelessly.
Even putting aside legal issues, it is very difficult to crack an encryption scheme in a short period of time. I personally do not have such ability, I can only hope that someone in the open source community has such ability.

**Reply by Ceqell** [reply]
Thanks for clarifying, may I ask why does qiyi doesn't want their smart timer to connect to cstimer wirelessly?

**Reply by dimikoan** [reply]
> qiyi doesn't want their smart timer to connect to cstimer wirelessly

Doesn't seem a good business idea. Perhaps more people will buy Gan Smart Timer instead.

**Comment by Quanloh** [comment]
They definitely wish to keep cubers on their app.
